portfolio/src/main/webapp/script.js

```diff
@@ -26,7 +26,13 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
-
+git 
 function openLink(link){
     window.open(link);
 }
+
+function getHello() {
+    fetch('/data').then(response => response.text()).then((greeting) => {
```

Comment:
Nit: 2 space indent.

It looks like Cloud Shell Editor uses 4 spaces by default, but you can change this to 2 in your preferences:

File > Settings > Preferences > Editor > tabSize

Reply:
Thanks, its a habit for me to use tab, that will definitely help with my spacing errors.